dask/dataframe/io/parquet/core.py

```diff
@@ -260,12 +254,12 @@ def read_parquet(
         ``"precache_options"`` key. Also, a custom file-open function can be
         used (instead of ``AbstractFileSystem.open``), by specifying the
         desired function under the ``"open_file_func"`` key.
-    engine : {'auto', 'pyarrow', 'fastparquet'}, default 'auto'
+    engine : {'auto', 'pyarrow'}
         Parquet library to use. Defaults to 'auto', which uses ``pyarrow`` if
-        it is installed, and falls back to ``fastparquet`` otherwise.
+        it is installed, and falls back to the deprecated ``fastparquet`` otherwise.
```

Comment:
nit: Should we at least mention somewhere that `fastparquet` does not support _all_ functionality?

Reply:
Q: which functionality does it not support? You could have asked?

Reply:
@martindurant: I've merely referred to the comments about functionality not supported within Dask's read parquet that have been removed in this PR. See the removed lines above this comment for more information.

Reply:
OK I understand. You mean that fastparquet as a backend in dask does not any more support all dask functionality (although fastparquet itself does). 

I am disappointed in how this issue (the whole PR) was handled without any notice to me, perhaps my comment above was not the right place for it. 

Nevertheless, let me state for the record: that fastparquet has remained remarkably stable, small and performant for a long time, while keeping up with all upstream changes in pandas. I am sorry to see it dropped like this.